### PR TITLE
Document how to get ssh access into CD testnets

### PR DESCRIPTION
**Pull request by mvines**

### CODE REVIEW

File: net/ssh.sh [hunk 1]
```diff
@@ -43,21 +43,26 @@ if [[ -n "$ipAddress" ]]; then
   exec ssh "${sshOptions[@]}" "$ipAddress" "$@"
 fi
 
+printNode() {
+  declare nodeType=$1
+  declare ip=$2
+  printf "  %-25s | $0 $ip tail -f /tmp/solana/=/$nodeType.log\n" "$0 $ip" #"$0 $ip"
```
Comment (rob-solana) [inline net/ssh.sh]: why do you printf $0 and $ip twice?

File: net/net.sh [hunk 2]
```diff
@@ -290,7 +299,7 @@ start() {
   echo "Leader deployment took $leaderDeployTime seconds"
   echo "Validator deployment (${#validatorIpList[@]} instances) took $validatorDeployTime seconds"
   echo "Client deployment (${#clientIpList[@]} instances) took $clientDeployTime seconds"
-  echo "Logs in $netLogDir:"
+  echo "Network start logs in $netLogDir:"
```
Comment (rob-solana) [inline net/net.sh]: what do you think of putting this stuff (and other like it) in /etc/motd?

File: net/ssh.sh (hunk 1 above)
Comment (mvines) [reply net/ssh.sh]: Here's what the resulting output looks like:
```
mvines@sol:~/ws/solana/net$ ./ssh.sh 
Leader:
  ./ssh.sh 104.196.252.185  | For logs run: ./ssh.sh 104.196.252.185 tail -f /tmp/solana/=/leader.log

Validators:
  ./ssh.sh 35.233.214.131   | For logs run: ./ssh.sh 35.233.214.131 tail -f /tmp/solana/=/validator.log
  ./ssh.sh 35.227.188.147   | For logs run: ./ssh.sh 35.227.188.147 tail -f /tmp/solana/=/validator.log
  ./ssh.sh 35.185.200.43    | For logs run: ./ssh.sh 35.185.200.43 tail -f /tmp/solana/=/validator.log
  ./ssh.sh 35.233.239.70    | For logs run: ./ssh.sh 35.233.239.70 tail -f /tmp/solana/=/validator.log
  ./ssh.sh 35.203.136.155   | For logs run: ./ssh.sh 35.203.136.155 tail -f /tmp/solana/=/validator.log
  ./ssh.sh 35.197.100.98    | For logs run: ./ssh.sh 35.197.100.98 tail -f /tmp/solana/=/validator.log
  ./ssh.sh 35.203.128.237   | For logs run: ./ssh.sh 35.203.128.237 tail -f /tmp/solana/=/validator.log
  ./ssh.sh 35.230.124.16    | For logs run: ./ssh.sh 35.230.124.16 tail -f /tmp/solana/=/validator.log
  ./ssh.sh 104.199.122.115  | For logs run: ./ssh.sh 104.199.122.115 tail -f /tmp/solana/=/validator.log
  ./ssh.sh 35.230.38.92     | For logs run: ./ssh.sh 35.230.38.92 tail -f /tmp/solana/=/validator.log

Clients:
  ./ssh.sh 35.230.77.249    | For logs run: ./ssh.sh 35.230.77.249 tail -f /tmp/solana/=/client.log
  ./ssh.sh 35.197.98.3      | For logs run: ./ssh.sh 35.197.98.3 tail -f /tmp/solana/=/client.log
```

File: net/net.sh (hunk 2 above)
Comment (mvines) [reply net/net.sh]: Yeah I like that idea.  I'd like to not add to this PR though ;)

**Comment by mvines** [comment]
Grumble #1012 (see https://buildkite.com/solana-labs/solana/builds/2964#66e38690-fd62-49a0-b47f-144a2f1f7a3c/903-960)

**Pull request merged**